libraries/ESP8266WiFi/examples/ConnectAsync/ConnectAsync.ino

```diff
@@ -0,0 +1,71 @@
+/*
+    This sketch tries to connect asincronously to a remote server.
```

Comment:
"asynchronously"

Reply:
Done in dd24fb4539ccc02105a420f215da8c8af46ddda9